src/Frontend/Assets.php

```diff
@@ -225,4 +235,9 @@ public function addFileSourceOverrides(array $fileSourceOverrides)
     {
         $this->fileSourceOverrides = array_merge($this->fileSourceOverrides, $fileSourceOverrides);
     }
+
+    public function addCustomLessFunctions(array $customFunctions)
+    {
+        $this->customFunctions = array_merge($this->customFunctions, $customFunctions);
+    }
```

Comment:
Why make this plural out of curiosity?

Reply:
Because you can pass an array with multiple functions in.

E.g.

```php
addCustomLessFunctions([
  'funcA' => ...,
  'funcB' => ...,
])

Reply:
Right, but why not

`addCustomLessFunction()->addCustomLessFunction()->addCustomLessFunction()`

Reply:
I think you're confusing this method for the extender one @askvortsov1 lol

Reply:
![](https://media0.giphy.com/media/4pMX5rJ4PYAEM/giphy.gif?cid=ecf05e47bey0caqfqgow45j88gwig5ofn2x5majt7bjp20j6&rid=giphy.gif&ct=g)